### PR TITLE
Hello World

### PR DESCRIPTION
**Pull request by ridaarif98**
This is a simple Hello-World project
   In this project, I add a markup file with a heading and paragraph with some description
   Add style for heading and paragraph in the stylesheet
   Add detail of the project in README file
   Integrate linters
   Resolve linetrs errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Hello-World
 This is simple Hello World Prject
-![screenshot](./hello.png)
+![screenshot](./hello1.png)
```
Comment (teekaytech) [inline README.md:3]: Your Readme looks promising, but you're having some placeholder texts (e.g. `This is an example of how you may give instructions on setting up your project locally`) in the  `Getting Started` section. I think you should replace the placeholder text with what actually resonates with this project. The text is there to guide you on what to do in the section :blush: 

Note that you can remove completely if any of the subsections (e.g. deployment) is not applicable to the project.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @ridaarif98 :wave:

__This is a brilliant effort from you guys. You've really done an amazing job so far. In this PR, my major highlights include__

- Liners are set up correctly with no errors :100: 
- PR title, summary and commit messages are descriptive :+1: 

__However, you need to make some required changes before moving on to the next assignment. Please check my comments below and fix them as appropriate.__

### **Required Changes** :recycle: 

- [ ] You might want to consider renaming your feature (new-feature) branch. Ensuring that your branch is named after a feature or action (e.g. 'hello_microverse', 'hello_microverse_feature' or 'added_hello_microverse_section') is one of the general requirements of this project. [This article](https://docs.github.com/en/github/administering-a-repository/managing-branches-in-your-repository/renaming-a-branch) might help in renaming your branch.
- [ ] You've done well to include the .gitignore file, listing test.md file. However, you have to list the `node_modules` directory as well, as it can be added by accident while setting up linters. Believe me, you don't want to commit/push your node_modules directory by mistake :smile: 
- [ ] Check the comment under the review.

### **Optional Changes** :zap: 
N/A


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

File: README.md (hunk 1 above)
Comment (ridaarif98) [reply README.md:3]: I try to fix things but when I change the branch name this pull request will be deleted

Comment (teekaytech) [reply README.md:3]: Hello @ridaarif98,

I do not mean to mislead you. I myself did not realize that renaming the branch will close the PR. Thanks for letting me know.

If renaming the branch will close/delete this PR, I suggest you should not bother to rename the branch. You can restore the branch, then go to the PR page for the project and mark the project as open.

**If you've already created and submitted a new PR, you can ignore this message.** However, you should take note of this against your subsequent projects.

**Pull request closed without merging**